README.md

```diff
@@ -102,7 +102,7 @@ Docker images hosted on [open-event-frontend repository](https://cloud.docker.co
     -   [ember inspector for chrome](https://chrome.google.com/webstore/detail/ember-inspector/bmdblncegkenkacieihfhpjfppoconhi)
     -   [ember inspector for firefox](https://addons.mozilla.org/en-US/firefox/addon/ember-inspector/)
 
-## Contributions Best Practices
+## Best Practices for Contributions
```

Comment:
Whatever changes u made does not relate to anything with the issue so Pls read the issue again carefully and do accordingly

Reply:
i remove this

Reply:
it is working fine for me now